tests/installation/decrypt_disks.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Module to descrypt encrypted disk
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "installbasetest";
+use utils;
+use testapi;
+
+sub run {
```

Comment:
yeah true. i should write this in the commit. the `installation/boot_encrypt` requires extra parameters to set in the test that i chose to avoid. Some of them are used in a way that cross and change behavior between modules. for instance it has different effect bettwen `separate_home` and and `installation/boot_encrypt`. i keep it simple and clean in that way.

Reply:
Set params is always more simple than create yet another test module that does exactly the same. That is wrong approach IMO unless you can explain better what are those problems with settings.

Reply:
The `unlock_if_encrypted` is not run unless i set `ENCRYPT=1`. But i dont use this and i dont want to. In additional, even if i set that, it breaks the `separate_home`. I cant remember where exactly now but it does.

Reply:
perfect then, as we need another module (i.e.: encrypt_separate_home). Otherwise, I'd suggest you gather more feedback for this.

Reply:
i am going to close this conversation. I believe that the variables in the test cause many issues and hacks that it should be avoided. Please reach me out of the PR to discuss it if you want.

Reply:
Sorry for unresolving, but I also believe that we should  reuse existing code and not add yet another module to handle this. It doesn't mean that we cannot change it to use test data or extract common part to the library methods.